cfme/tests/automate/test_method.py

```diff
@@ -749,3 +750,50 @@ def test_null_coalescing_fields(request, klass):
     )
 
     assert log.validate()
+
+
+@pytest.mark.tier(3)
+def test_automate_user_has_groups(request, appliance, custom_instance):
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1411424
+    This method should work:  groups = $evm.vmdb(:user).first.miq_groups
+    $evm.log(:info, "Displaying the user"s groups: #{groups.inspect}")
+
+    Bugzilla:
+        1411424
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: automate
+        startsin: 5.8
+    """
+    user, user_data = _users(request, appliance)
+
+    script = dedent(
+        f"""
+        group = $evm.vmdb(:user).find_by_name("{user[0].name}").miq_groups
+        $evm.log(:info, "Displaying the user's groups: #{{group.inspect}}")
+        """
+    )
+    instance = custom_instance(ruby_code=script)
+
+    with LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[f'.*{user_data[0]["group"]["description"]}.*'],
+    ).waiting(timeout=120):
+
+        # Executing automate method using simulation
```

Comment:
**Question:**
Extra Indentation .? or `simulate()` is under `with()`

Reply:
yeah. ```simulate``` under ```with LogValidator```.